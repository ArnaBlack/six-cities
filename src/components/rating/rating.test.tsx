import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Rating from './rating';

const mock = {
  hasValue: false,
  rating: 3.2,
};

it(`Rating renders correctly`, () => {
  const {
    hasValue,
    rating,
  } = mock;
  const tree = renderer
    .create(<Rating
      hasValue={hasValue}
      rating={rating}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
