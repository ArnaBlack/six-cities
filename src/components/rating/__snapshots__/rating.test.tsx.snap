// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating renders correctly 1`] = `
<div
  className="rating undefined"
>
  <div
    className="rating__stars undefined"
  >
    <span
      style={
        Object {
          "width": "60%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
</div>
`;
